src/Transition/CreateNewIDFUsingRulesV24_1_0.f90

```diff
@@ -429,14 +429,22 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
                 nodiff=.false.
 
                 ! read in 8 reference value for the effectiveness at 75% and 100%
-                HxEffectAt75Airflow(1) = TRIM(InArgs(6))    ! Sensible Effectiveness at 75% Heating Air Flow
-                HxEffectAt75Airflow(2) = TRIM(InArgs(7))    ! Latent Effectiveness at 75% Heating Air Flow
-                HxEffectAt75Airflow(3) = TRIM(InArgs(10))   ! Sensible Effectiveness at 75% Cooling Air Flow
-                HxEffectAt75Airflow(4) = TRIM(InArgs(11))   ! Latent Effectiveness at 75% Cooling Air Flow
-                HxEffectAt100Airflow(1) = TRIM(InArgs(4))   ! Sensible Effectiveness at 100% Heating Air Flow
-                HxEffectAt100Airflow(2) = TRIM(InArgs(5))   ! Latent Effectiveness at 100% Heating Air Flow
-                HxEffectAt100Airflow(3) = TRIM(InArgs(8))   ! Sensible Effectiveness at 100% Cooling Air Flow
-                HxEffectAt100Airflow(4) = TRIM(InArgs(9))   ! Latent Effectiveness at 100% Cooling Air Flow
+                ! Sensible Effectiveness at 75% Heating Air Flow
+                HxEffectAt75Airflow(1) = GetFieldOrIDDDefault(InArgs(6), FldDefaults(6))
```

Comment:
So FldDefaults will always be populated with meaningful or blank values?  I've never used that array so just confirming.